### DISCUSSION TITLE
GO terms in GAF file

### DISCUSSION THREAD

**Discussion by JieWu2012** [geneontology/helpdesk#433] [General]
I can not find the GO term (GO:0050789 ) in the human GAF file that I downloaded from (http://current.geneontology.org/annotations/). But when I search the term in (http://geneontology.org/), I can find it. Besides, the amounts of genes that belongs to specific term in GAF file is very different from the records returned from the search in (http://amigo.geneontology.org/amigo/landing). Can someone tell me why?  Does the GAF file contain information of all GOterm-gene pairs?

**Comment by balhoff** [comment]
@JieWu2012 I believe you are seeing the fact that the ontology is a hierarchy of terms. The search in AmiGO is configured to return all data annotated to the term you search, as well as annotated to any descendants of the term.
